requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==2.2.*
+numpy==2.0.*
```

Comment:
@dependabot mistake, actually a DOWNgrade to https://github.com/numpy/numpy/releases/tag/v2.0.2 from August 2024

See
* https://github.com/numpy/numpy/releases
* https://github.com/numpy/numpy/tags

Reply:
@dependabot recreate